packages/app/src/app/pages/Sandbox/Editor/Workspace/items/GitHub/GitHub.tsx

```diff
@@ -38,7 +38,12 @@ export const GitHub = observer(() => {
     )
   ) : (
     <>
-      <Description margin={1} top={0}>
+      <Description
```

Comment:
Hmm, why not passed as args?

Reply:
TypeScript gave an error when passing them as normal args, since it's a `styled.div`

Reply:
Hmm, you can fix that by defining which props the div gets. So it would be

```
styled.div<{ margin?: number; top?: number; ... }>`
`
```

It probably defers the props from the `Margin` component, so probably

```
import Margin, { MarginProps } from 'place-to-margin';

styled(Margin)<MarginProps>`
`

```

I have to mention that I didn't look at the code, this seems like the current situation.

I think that's a cleaner solution.

Reply:
Will do 🙂

Reply:
@CompuIves This is the implementation of `Description`
```js
export const Description = styled.div`
  margin: 0.5rem 1rem;
  line-height: 1.4;
  font-size: 0.875rem;
  color: ${props =>
    props.theme.light ? 'rgba(0, 0, 0, 0.8)' : 'rgba(255, 255, 255, 0.8)'};
`;
```

Don't know what  you mean by the defer-thing tbh

Reply:
Ahh, well, that means that we can just drop the `margin` and `top` anyway, they were not used anyway! The power of TypeScript :).

Reply:
Yeah that's what I was thinking too, so should I remove the `css` or just delete it all since it wasn't used anyway? 🤔

Reply:
Yeah you can remove the `css` and `margin` and `padding` attrs.

Reply:
OK will do 👍